src/napari_tomoslice/_constants.py

```diff
@@ -42,3 +42,11 @@
 - press 'r' at desired edge of sphere to change the radius
 - {_ALT} + click again to move the sphere 
 """
+
+DIPOLE_ANNOTATOR_HELP_TEXT = f"""
+- press 'n' to start a new dipole
+- {_ALT} + click to set the position of the new dipole
+- press 'v' to set the direction of the dipole
```

Comment:
I've been second guessing this - not sure why but it didn't feel intuitive when I was playing with it. How do we feel about 'd' for direction instead? 🙂 if you agree I'll update n3d and push a release

Reply:
Yes, I like that!